base/markdown/render/plain.jl

```diff
@@ -42,6 +42,8 @@ function plain(io::IO, md::HorizontalRule)
     println(io, "–" ^ 3)
 end
 
+plain(io::IO, md) = writemime(io, "text/plain", md)
```

Comment:
There is a `writemime` method that converts the string argument to a `MIME` object.